routing/pathfind.go

```diff
@@ -484,7 +484,11 @@ type findPathParams struct {
 
 	// bandwidthHints is an optional map from channels to bandwidths that
 	// can be populated if the caller has a better estimate of the current
-	// channel bandwidth than what is found in the graph.
+	// channel bandwidth than what is found in the graph. If set, it will
+	// override the capacities and disabled flags found in the graph when
+	// doing path finding. In particular, it should be set to the current
```

Comment:
>If set, it will override the disabled flags found in the graph.

It isn't `bandwidthHints` that overrides the disabled flag, is it? Only the fact that a channel is local, overrides the disabled flag?

Reply:
Yes, you're right. I forgot to update the comment with the latest revision. I was contemplating whether it would also make sense to let the bandwidth hint override disabled flags for non-local channels, but figured for now that is not needed (as we don't use it).

Reply:
Updated!